daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6/daml_lf.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
```

Comment:
Given that the non-dev DAML-LF proto files are supposed to be frozen, I’m not sure it makes sense to update the copyright header. I thought we even have a test that the hash of these files doesn’t change.

Reply:
Indeed. Reverting that. I assume that's only for numbered versions, i.e. not the dev ones?

Reply:
Yeah the dev proto files change.